README.md

```diff
@@ -21,12 +24,22 @@ One common setup is to create a directory for a manuscript that contains both th
 Under this setup, you can run the Manubot using:
 
 ```sh
-manubot \
+manubot process \
   --content-directory=content \
   --output-directory=output
 ```
 
-See `manubot --help` for documentation of all command line arguments.
+See `manubot process --help` for documentation of all command line arguments.
+
+### Cite
+
+`manubot cite` is a command line utility to create CSL JSON items for one or more citations.
+Citations should be in the format `source:identifier`.
+For example, the following example generates CSL JSON for four references:
+
+```sh
+manubot cite doi:10.1098/rsif.2017.0387 pmid:29424689 pmcid:PMC5640425 arxiv:1806.05726
+```
```

Comment:
Perhaps it is obvious but adding
> See manubot cite --help for documentation of all command line arguments.

or a similar message could help users discover the `--file` option.

Reply:
I decided we should just put the output of help in the README. I think it's useful for people to see the full help message without having to install the program. It also encourages us to document the argparse and avoid duplicating documentation.

Reply:
Yes, I like that solution.